app/main.py

```diff
@@ -1,8 +1,30 @@
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
 
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+        self.__class__.people[name] = self
```

Comment:
It's common practice to refer to class attributes directly by the class name rather than through `self.__class__`. This makes the code more readable and clear about which attribute is being accessed

When dealing with inheritance, using `self.__class__` to reference a class attribute can be beneficial because it ensures that the attribute of the current class is used, not the one from the parent class